cmd/compute-domain-kubelet-plugin/computedomain.go

```diff
@@ -67,6 +66,10 @@ type ComputeDomainDaemonSettings struct {
 	nodesConfigPath string
 }
 
+func (c Config) ComputeDomainDaemonSettingsRoot() string {
+	return filepath.Join(c.DriverPluginPath(), "domains")
+}
+
```

Comment:
I don't think we should add a method for this (or others like this) -- there should just be more variables added to the config to hold these values.

Reply:
instead of adding to config, how about making them hardcoded in a var in the function itself like we have for setting NvidiaCdihook

Reply:
`DriverPluginPath()` shouldn't be a function either.